google/cloud/security/inventory/inventory_loader.py

```diff
@@ -185,8 +185,43 @@ def _run_pipelines(pipelines):
                          exc_info=True)
             pipeline.status = 'FAILURE'
         run_statuses.append(pipeline.status == 'SUCCESS')
+    _postprocess_statuses(pipelines, run_statuses)
```

Comment:
Should the result that is returned from ```_postprocess_statuses()``` be used here?  Otherwise, what is the state of ```run_statues``` and ```pipelines``` here?

Reply:
`_postprocess_statuses()`  modifies the actual data/parameters passed to it.

Reply:
See e.g. http://cryptroix.com/2016/10/25/python-call-object/

Reply:
Ah, interesting; good thing to learn.  :)  So, yes, in this case, this is fine and no change necessary.